src/Core/Settings.h

```diff
@@ -105,7 +105,7 @@ class IColumn;
     M(UInt64, s3_retry_attempts, 10, "Setting for Aws::Client::RetryStrategy, Aws::Client does retries itself, 0 means no retries", 0) \
     M(UInt64, s3_request_timeout_ms, 3000, "Idleness timeout for sending and receiving data to/from S3. Fail if a single TCP read or write call blocks for this long.", 0) \
     M(Bool, enable_s3_requests_logging, false, "Enable very explicit logging of S3 requests. Makes sense for debug only.", 0) \
-    M(String, s3queue_default_zookeeper_path, "/s3queue/", "Default zookeeper path prefix for S3Queue engine", 0) \
+    M(String, s3queue_default_zookeeper_path, "/clickhouse/s3queue/", "Default zookeeper path prefix for S3Queue engine", 0) \
```

Comment:
Why this path is any better? Just interesting.

Reply:
Alexey proposed to change it this way.

Reply:
Sometimes, people use ZooKeeper not only for ClickHouse - that's why we have `/clickhouse/tables` in many examples. It's reasonable to have `s3queue` at the same level as `tables`. If `tables` are inside `/clickhouse` and s3queue isn't - it will stand out. Also, I expect that people will naturally use the default path in most cases.